src/Components/ResultList.tsx

```diff
@@ -50,6 +58,11 @@ export default class ResultList extends React.Component {
     this.setState(this.headlessResultList.state);
   }
 
+  getDate(result: Result) {
+    let date: Date = new Date(result.raw.date);
```

Comment:
You should use `const` here.

In general, `const` doesn't mean that the variable is immutable, it only means that you won't be reassigning it (i.e., you can't do `x = "something"` or `x += "something else"` once you've declared `x` as a `const`).

So for instance if you do:

```javascript
let x = { foo: "bar" };
let y = { foo: "bar" };
```

then doing:

```javascript
x.foo = "hello";
y.foo = "world";
```

will work

and so will

```javascript
x = "foo" 
```

however, trying to do

```javascript
y = "foo";
```

will return `Uncaught TypeError: Assignment to constant variable.`